### DISCUSSION TITLE
ios16.1 could not release the storage

### DISCUSSION THREAD

**Discussion by Kabomani** [localsend/localsend#909] [Discussion (English, International)]
I'm using IOS 16.1, and I found that after deleting the files received or sent under the Files app, the storage used in the system will still count. I have to delete the app from my phone to release this amount of storage. 
I haven't found any discussions about storage problems before, if there is one please dismiss mine discussion and let me know the solution, thanks a lot

**Comment by Tienisto** [comment]
Hello, I think this is a known problem. However, I cannot reproduce this on my iphone simulator. I will check this on a real device in the next days

**Reply by Tienisto** [reply]
@Kabomani Can you write down the concrete repoduction steps? What file picker did you use? Did you pressed on "Media" or on "File"? This would help me to fix the bug
